iridauploader/model/directory_status.py

```diff
@@ -248,6 +254,8 @@ def __init__(self, sample_name, project_id, uploaded=False):
             """
             self._sample_name = sample_name
             self._project_id = project_id
+            if type(uploaded) is str:
+                uploaded = uploaded.lower() in ['true', '1', 't', 'y', 'yes']
```

Comment:
Can you add a comment on what's going on with this section?  It's not clear what it's doing.

Reply:
Added a comment.
The `X in Y` keyword function returns a boolean checking if `X` is in list `Y`, which is used in this standard way of changing a string with a boolean value to an actual boolean.